app/services/travis_script.rb

```diff
@@ -0,0 +1,14 @@
+require 'uri'
+require 'net/http'
+require 'json'
+
+uri = URI.parse("https://api.travis-ci.org/repo/ruby-bench%2Fruby-bench-docker/requests")
```

Comment:
It's just the same script I used from [here](https://docs.travis-ci.com/user/triggering-builds). So kept it that way.